### ISSUE TITLE
REPORT DEVICE SPECIFIC BUGS ONLY. LineageOS bugs should be reported upstream.

### ISSUE THREAD

**Issue by vishnunaini** [vishnunaini/lineage-surnia-releases#11]

**Comment by felipewinchester** [comment]
Hello, I'm having a problem at the time of making the non-mobile phone Moto and 2015 Vollte surnia.
I can not do the root, but when the phone is starting, and I do not know. And when I enter (use data, and click on the three side markers to appear a "network" option to access as settings), a "com.android.phone stopped" error occurs and thus has an option to open the app again. But re-opening gives the same error
I would like to root in the mobile phone, but I will have to wait until I fix the problem.

I loved Rom's work and thank you for your attention!



Português

Olá, eu estou com um problema na hora de fazer a raiz não celular Moto e 2015 Vollte surnia.
Eu não consigo fazer a raiz, porem, quando o telefone está começando, e eu não sei. E quando entro em (uso de dados, e clico nas três bolinhas laterais para aparecer uma opção "rede" para acessar como configuracoes), ocorre um erro "com.android.phone parou" e assim tem uma opção de abrir o app denovo. Mas voltando a abrir dá o mesmo erro
gostaria de fazer root no me celular, mas vou ter que esperar ate corrigir o problema.

Adorei o trabalho da Rom e agradeço des de a pela atençao !

**Comment by vishnunaini** [comment]
Root is not available in this ROM. But if you still want to root it Download chainfire root from here https://plus.google.com/+Chainfire/posts/27xToSqi5em . Flash it through your recovery.

Cell network issue: 
The settings app sometimes misbehaves in foreign models. I can fix it if I have the logs but I don't because I do not have your particular model on hand to test it. I only have an XT1521 but you might be using an XT 1524 .

**Comment by vishnunaini** [comment]
Please use https://github.com/vishnunaini/lineage-surnia-releases/releases/download/20170118/lineage-14.1-20170118-10-vishnunaini-surnia.zip build instaed of the 20170120 build.

**Comment by felipewinchester** [comment]
Hi, thanks for the support. I am informing that I was able to do Root in the version used. I ended up using TWRP to install only the System, and then I installed the SuperSU from an outside site. I think the files for Root were not compatible.
My doubt now, I'm not sure if it's a bug.

When I put apps like Chrome or Yandex to replay YouTube videos, the system will not let you play outside the app or even on the lock screen. In Android 6.1 it was used to use this function that chrome and yandex leaves.
I would like to ask for updates, to get straight to the phone. I do not know if in the future, the company plans to put this, but it would be a great resource, which would help many.

Thanks for the attention, and I'm here to compliment the great work! This Rom is very perfect. And especially the Battery that on the bike And lasted almost 1 whole day, for the start of a great project, I can say that is the best Rom of existing Android!

Apart from this problem, I'm very satisfied with Rom and very happy to see that Lineage OS is growing and doing a great job!

Sorry for the inconvenience and thank you very much for your attention.





Portuguese.

Olá, obrigado pelo suporte. Estou informando que consegui fazer o Root na versão utilizada. Acabei usando o TWRP para instalar só o Sistema, e em seguida, Instalei o SuperSU de um site exterior. Acho que os arquivos para Root, não estavam compativeis.
Minha duvida agora, não tenho certeza se é um bug.

Quando coloco aplicativos como (Chrome ou Yandex), para reprodizir videos do YouTube, o sistema não deixa reproduzir fora do aplicativo ou ate mesmo na tela de bloqueio. No android 6.1 dava para ultilizar essa função que o chrome e o yandex deixa.
Gostaria de pedir que atualizações, chegasse direto no celular. Não sei se no futuro, a empresa pensa colocar isso, mas seria um otimo recurso ,que ajudaria muitos.

Obrigado pela atenção, e estou aqui para elogiar mais o grande trabalho! essa Rom está muito perfeita. e principalmente a Bateria que no moto E durou quase 1 dia todo, para o inicio de um grande projeto, eu posso falar que é a melhor Rom de Android existente !  

Fora esse problema, eu estou muito satisfeito com a Rom e muito feliz de ver que a Lineage OS esta crescendo e fazendo um otimo trabalho !

Desculpe pelo inconveniente e agradeço muito pela atenção.

**Comment by vishnunaini** [comment]
I cannot provide direct OTA updates because I don't have the financial
means to maintain an OTA server. Providing OTA updates requires the ROM to
be signed with private release keys for security reasons, but most
recoveries will break with private keys.

**Comment by 003cyriac** [comment]
You rock bro.. Very improvising updates.. Keep up the good work.. Will help you recognise some bugs while using it in my day to day life..👍

**Comment by felipewinchester** [comment]
Hello, I have not said anything about the rom after your response, well, it is great and updated when I saw your response. But before upgrading I had already managed to get Root.

I would like to thank the support, and now compriedi, the problems with the updates in OTA.

(I do not know if you know, but in the Pre-release I just found a problem.) I'm going to go back to stable version and wait for the next stable!

Congratulations on the great work, and I do not change LineageOS for nothing.

Thank you for the support and the amazing Rom!

Long life to LineageOS.



Português

Olá, não falei nada sobre a rom depois da sua resposta, bem, ela esta otima e atualizei quando vi sua resposta. mas antes de atualizar eu ja tinha conseguido colocar o Root. 

gostaria de agradeçer o apoio, e agora compriedi, o problemas com as atualizações em OTA.

(não sei se você sabe, mas no Pre-release so achei um probleminha. a camera do celular não funciona) vou voltar para versao estavel e esperar a proxima estavel! 

Parabens pelo otimo trabalho, e não troco a LineageOS por nada. 

obrigado pelo suporte e pela incrivel Rom! 

Vida longa ao LineageOS.
